pom.xml

```diff
@@ -55,6 +55,13 @@
 			<groupId>com.github.ben-manes.caffeine</groupId>
 			<artifactId>caffeine</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
You probably now this, but there is a dept44-release ongoing. When this is released, this can be removed (since SB 2.7.10 is bundled with that release)